frontend/src/app/main/data/workspace/colors.cljs

```diff
@@ -518,9 +518,11 @@
                   (if (some? gradient)
                     (let [stop  (or (:editing-stop state) 0)
                           stops (mapv split-color-components (:stops gradient))
-                          type  (case (:type gradient)
-                                  :linear :linear-gradient
-                                  :radial :radial-gradient)]
+                          type  (if (:type gradient)
```

Comment:
The if is redundant. Use the case with gradient type and in the "else" branch use (:type shape)

Reply:
Thanks!